AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Benchmark_GPU.py

```diff
@@ -15,16 +16,76 @@
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.parametrize('rhi', ['dx12', 'vulkan'])
+def filebeat_service_running():
+    """
+    Checks if the filebeat service is currently running on the OS.
+    :return: True if filebeat service detected and running, False otherwise.
+    """
+    result = False
+    try:
+        filebeat_service = psutil.win_service_get('filebeat')
+        filebeat_service_info = filebeat_service.as_dict()
```

Comment:
Does this need an if around the use of filebeat_service? can it be null?

Reply:
if the `psutil.win_service_get()` call fails it will trigger a `psutil.NoSuchProcess` exception then exit into the exception block which will return `False` so it should be fine